unobstructed_lanechange/include/unobstructed_lanechange.h

```diff
@@ -80,6 +80,8 @@ namespace unobstructed_lanechange
                                                 double max_starting_downtrack,
                                                 const carma_wm::WorldModelConstPtr& wm,const cav_msgs::VehicleState& state);
             int getNearestRouteIndex(lanelet::BasicLineString2d& points, const cav_msgs::VehicleState& state);
+
+            int getNearestPointIndex(const std::vector<lanelet::BasicPoint2d>& points, const cav_msgs::VehicleState& state) const;
```

Comment:
@lewisid Are these changes to unobstructed_lanechange intentional or from a bad merge? If the intentional, I'll need to get Anish to review.